module/icp/asm-x86_64/modes/aesni-gcm-x86_64.S

```diff
@@ -50,14 +50,16 @@
 #define _ASM
 #include <sys/asm_linkage.h>
 
+/* Windows userland links with OpenSSL */
+#if !defined (_WIN32) || defined (_KERNEL)
+
 .extern gcm_avx_can_use_movbe
 
 .text
 
 #ifdef HAVE_MOVBE
-.type	_aesni_ctr32_ghash_6x,@function
-.align	32
-_aesni_ctr32_ghash_6x:
+.align 32
+FUNCTION(_aesni_ctr32_ghash_6x)
```

Comment:
ENTRY_ALIGN()? Looks like an old version?

Reply:
This is a special case, the function `aesni_ctr32_ghash_6x` in not called from C, so this PR doesn't need to `ASMABI` it, and it is not defined as `globl` so no need for `ENTRY_NP`.

it is defined as:

```
.type	_aesni_ctr32_ghash_6x,@function
.align	32
_aesni_ctr32_ghash_6x:
```

and Windows don't handle the `.type function` so I made a macro `FUNCTION()` to put that in for Linux, and empty for Windows.
I left the `align` as if for now, when I do macOS, I'll do a PR to unify the aligns.

Reply:
Indeed, I should've noticed it. Sorry for the noise.